### PR TITLE
Update Google Chrome version

### PR DESCRIPTION
**Pull request by heavenshell**
# TL;DR

Google Chrome version was updated

### CODE REVIEW

**Comment by dkimura** [comment]
```
shoota [5:53 PM]: これ毎回バージョン上げないとならない？
shoota [5:53 PM]: ltsとか指定したい
```

ちょっと調べる必要あるかな。

File: Dockerfile [hunk 1]
```diff
@@ -40,7 +40,7 @@ RUN curl -sS https://dl-ssl.google.com/linux/linux_signing_key.pub | apt-key add
     echo "deb [arch=amd64] http://dl.google.com/linux/chrome/deb/ stable main" | tee /etc/apt/sources.list.d/google.list && \
     apt-get update && \
     apt-get install --no-install-recommends -y \
-      google-chrome-stable=66.0.3359.139-1 && \
+      google-chrome-stable=66.0.3359.170-1 && \
```
Comment (shoota) [inline Dockerfile]: can not use `google-chrome-stable` ?

Comment (shoota) [reply Dockerfile]: i check this.

```diff
@@ -40,7 +40,7 @@ RUN curl -sS https://dl-ssl.google.com/linux/linux_signing_key.pub | apt-key add
     echo "deb [arch=amd64] http://dl.google.com/linux/chrome/deb/ stable main" | tee /etc/apt/sources.list.d/google.list && \
     apt-get update && \
     apt-get install --no-install-recommends -y \
-      google-chrome-stable=66.0.3359.139-1 && \
+      google-chrome-stable && \
     apt-get clean && \
     rm -rf /var/lib/apt/lists/* && \
     google-chrome --version
```
and output is this.
> Google Chrome 66.0.3359.170

Comment (heavenshell) [reply Dockerfile]: Thanks I'll fix this!

**Pull request merged**